### DISCUSSION TITLE
Any info on roadmap/relase schedule for just?

### DISCUSSION THREAD

**Discussion by noahehall** [just-js/just#33] [General]
whats state is the project currently in? when can we expect an alpha/beta/GA release etc?

**Comment by noahehall** [comment]
https://github.com/just-js/just/issues/31

**Comment by KabilanV** [comment]
How is the progress of Just?

**Comment by billywhizz** [comment]
i am having a look at it again now, after a long break due to personal circumstances. let me know if you have any thoughts/ideas/suggestions. closing any non-important issues and discussions and will be writing up some thoughts in coming weeks.
